Code/Source/Utilities/ROS2Names.h

```diff
@@ -5,26 +5,35 @@
  * SPDX-License-Identifier: Apache-2.0 OR MIT
  *
  */
-
 #pragma once
 
-#include <AzCore/std/string/string.h>
 #include <AzCore/Outcome/Outcome.h>
 #include <AzCore/RTTI/TypeInfo.h>
+#include <AzCore/std/string/string.h>
 
 namespace ROS2
 {
+    //! Utility class for handling ROS2 naming rules
     class ROS2Names
     {
     public:
+        //! Joins namespace and the given name
```

Comment:
Nitpicking: sometimes you are placing a dot at the end of the sentence and sometimes you don't. Some more consistency would be great :)

Reply:
Dots all the way. Dots are actually required if there is more than one sentence (even in separate lines) to divide into Brief/Description.